docs/quickstart.rst

```diff
@@ -5,9 +5,9 @@ Quickstart
 
 In this section, we provide a simple guide for running a Kafka cluster along with all of the other Confluent Platform components.  By the end of this quickstart, you will have successfully installed and run a simple deployment including each component with Docker.
 
-In order to keep things simple, this quickstart guide is limited to a single node Confluent Platform cluster.  For more advanced tutorials, you can refer to the `tutorials section <tutorials.html>`_ of this documentation.
+In order to keep things simple, this quickstart guide is limited to a single node Confluent Platform cluster.  For more advanced tutorials, you can refer to the :ref:`tutorials section <tutorials_overview>` of this documentation.
```

Comment:
Same issue here--link only takes you to the main doc landing page here: https://docs.confluent.io/current/

Reply:
Same question as above, I don't think you have a way to test this unless you build locally.

Reply:
@JoyceFee1 see my previous comments